src/components/SearchResults.vue

```diff
@@ -80,12 +86,14 @@ export default {
     modeQuery() {
       let query = "";
 
-      if (this.mode === "ALL") {
+      if (this.mode === "all") {
```

Comment:
This is probably worth refactoring to a switch statement now - alternatively, you could define a key/value map with a fallback? e.g.

```
const queryMap = {
  'all': '',
  'ux': 'label:type/ux',
  ..
}

return queryMap[this.mode] || 'Unknown';
```

Reply:
Nice - done